css/main.scss

```diff
@@ -347,6 +347,13 @@ i:hover {
   }
 }
 
+.header {
+  background-color: #e4f7fd;
+  padding: 10px 0;
+  font-size: 12px;
+  text-align: center;
+}
```

Comment:
I think we should use a white background and make the content right-aligned as it is in the footer.